packages/labs/analyzer/src/lib/javascript/classes.ts

```diff
@@ -19,7 +19,7 @@ export const getClassDeclaration = (
   _programContext: ProgramContext
 ): ClassDeclaration => {
   return new ClassDeclaration({
-    name: declaration.name?.text,
+    name: declaration.name?.text ?? '',
```

Comment:
This change makes all `Declaration`s now extend a common base class that has `name: string` (before some were `string | undefined`). Ultimately I think this is correct since I don't think we need any anonymous declarations in the model -- if an anonymous class/function expression is assigned to a const, we'll use that name in the declaration (I have later versions that do that).  Once we support that, this will go away, so I'll add a TODO.

Reply:
We could have something like:

```ts
customElements.define('my-element, class extends LitElement { ... });
```

Rare, but possible.